.travis.yml

```diff
@@ -12,16 +12,33 @@ compiler:
     - gcc
 
 env:
-    # Test with our internal pypi mirror
+    # Test with our internal pypi mirror, locale en-US
     - PIP_FIND_LINKS=http://pypi.pub.build.mozilla.org/pub
       PIP_NO_INDEX=1
       DISPLAY=$(if [[ $TRAVIS_OS_NAME = 'linux' ]]; then echo ':99'; fi)
       MOZ_XVFB=$(if [[ $TRAVIS_OS_NAME = 'linux' ]]; then echo 1; fi)
       PIP_STRICT_VERSIONS=--strict
-    # ... and with the official pypi website
+      LOCALE='en-US'
```

Comment:
Lets add this as the first item right before PIP_FIND_LINKS so it will be easier to discover on the Travis CI build. If we don't do that it's not clearly visible why we have to many different jobs.